src/jit/instrsxarch.h

```diff
@@ -369,8 +370,10 @@ INST3( pcmpeqb,     "pcmpeqb"     , 0, IUM_WR, 0, 0, BAD_CODE,     BAD_CODE,
 INST3( pcmpgtb,     "pcmpgtb"     , 0, IUM_WR, 0, 0, BAD_CODE,     BAD_CODE,      PCKDBL(0x64))   // Packed compare 8-bit signed integers for greater than
 
 INST3( pshufd,      "pshufd"      , 0, IUM_WR, 0, 0, BAD_CODE,     BAD_CODE,      PCKDBL(0x70))   // Packed shuffle of 32-bit integers
+INST3( pshufhw,     "pshufhw"     , 0, IUM_WR, 0, 0, BAD_CODE,     BAD_CODE,      SSEFLT(0x70))   // Shuffle the high words in xmm2/m128 based on the encoding in imm8 and store the result in xmm1.
+INST3( pshuflw,     "pshuflw"     , 0, IUM_WR, 0, 0, BAD_CODE,     BAD_CODE,      SSEDBL(0x70))   // Shuffle the low words in xmm2/m128 based on the encoding in imm8 and store the result in xmm1.
 INST3( pextrw,      "pextrw"      , 0, IUM_WR, 0, 0, BAD_CODE,     BAD_CODE,      PCKDBL(0xC5))   // Extract 16-bit value into a r32 with zero extended to 32-bits
-INST3( pinsrw,      "pinsrw"      , 0, IUM_WR, 0, 0, BAD_CODE,     BAD_CODE,      PCKDBL(0xC4))   // packed insert word
+INST3( pinsrw,      "pinsrw"      , 0, IUM_WR, 0, 0, BAD_CODE,     BAD_CODE,      PCKDBL(0xC4))   // Move the low word of r32 or from m16 into xmm at the word position specified by imm8
```

Comment:
nit: The comment at the end-of-line generally looks to be the Arch Manual "long name", rather than actual descriptive text. e.g. `PINSRW—Insert Word`, so the comment should be `// Insert Word` or `PSHUFLW—Shuffle Packed Low Words`, so the comment should be `// Shuffle Packed Low Words`

Reply:
Actually there are both formats used - in majority of cases this is descriptive text like above on line 375

Reply:
Personal preference is on the "long name", since attempting to describe the instruction behavior can quickly become too complicated, especially when multiple encodings are supported.

Reply:
I agree with @tannergooding on this - I suspect that many of the comments that don't conform to this are older. In any case, going forward I think that the Arch Manual "long name" is the way to go.

Reply:
OK will fix and keep all comments in feature in "long name" format.